### PR TITLE
make defmt attributes forward input attributes

### PR DESCRIPTION
**Pull request by japaric**
work towards fixing #252 

- [x] reject {panic_handler,timestamp} + ( `#[export_name]` OR `#[no_mangle]`)

(I was expecting `#[timestamp] #[inline(always)] fn foo() -> u64 { 0 }` to fail at link time but it works ... perhaps export_name undoes inlining?)

### CODE REVIEW

**Review by Lotterleben — APPROVED** [review]

File: macros/src/lib.rs [hunk 1]
```diff
@@ -95,8 +121,13 @@ pub fn panic_handler(args: TokenStream, input: TokenStream) -> TokenStream {
             .into();
     }
 
+    let attrs = &f.attrs;
+    if let Err(e) = reject_attributes("panic_handler", attrs, &["export_name", "no_mangle"]) {
```
[the remark this answers is not in the document]
Comment (japaric) [reply macros/src/lib.rs:125]: good call

**Comment by japaric** [comment]
bors r=Lotterleben

**Pull request merged**